recipe/meta.yaml

```diff
@@ -1,46 +1,55 @@
+{% set name = "pyepics" %}
 {% set version = "3.4.0" %}
+{% set sha256 = "a2913ee5fdab9dfb3d50c0ccb83f7129902e6fdc7ce9c4c1cc9d0d4b2f9c0438" %}
+
 package:
-    name: pyepics
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    url: https://github.com/pyepics/pyepics/archive/{{ version }}.tar.gz
-    fn: pyepics-{{ version }}.tar.gz
-    patches:
-      - libca.patch
-build:
-    number: 1
-    script: |
-      export NOLIBCA=1
-      echo "Using LIBCA from $PYEPICS_LIBCA"
-      rm -rf epics/clibs
-      python setup.py install --single-version-externally-managed --record=record.txt
+   fn: {{ name }}-{{ version }}.tar.gz
+   url: https://github.com/pyepics/{{ name }}/archive/{{ version }}.tar.gz
+   sha256: {{ sha256 }}
 
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True # [py<36]
+  patches:
+      - libca.patch
```

Comment:
@buijennifer, this patch block should belong to the `source` section, not `build`.